pkg/aws/awscontrolplane/validate_awscontrolplane.go

```diff
@@ -83,7 +93,7 @@ func (v *Validator) Validate(request *admissionv1.AdmissionRequest) (bool, error
 
 	}
 
-	return controlPlaneLabelMatches && azAllowed && azCountAllowed && azReplicaMatches && instanceTypeAllowed, nil
+	return controlPlaneLabelMatches && azAllowed && azCountAllowed && azUnique && azOrderKept && azReplicaMatches && instanceTypeAllowed, nil
```

Comment:
I am curious for how long you think this design is sustainable. 😅

Reply:
Actually we might want to change it and just return true at that point. If one validation check throws an error we return false earlier anyway. I'll put it in the list for the refactoring.